index.html

```diff
@@ -470,40 +450,42 @@ <h3>
             `watchPosition()` method
           </h3>
           <p>
-            The <dfn>watchPosition()</dfn> method takes one, two or three
-            arguments. When called, immediately return a long value that
-            uniquely identifies a <a>watch process</a> and continue
-            asynchronously.
+            When <dfn>watchPosition()</dfn> is invoked, the user agent MUST:
           </p>
           <ol>
+            <li>Let |watchId:long| be {{long}} that uniquely identifies a
+            <a>watch process</a>.
+            </li>
+            <li>Return |watchId| and continue [=in parallel=].
+            </li>
             <li>If the <a>environment settings object</a> is a <a>non-secure
-            context</a> or the <a>current settings object</a>'s [=environment
-            settings object / responsible document=] is not [=allowed to use=]
-            the "geolocation" feature:
-              <ol>
-                <li>If |errorCallback| is not null, invoke the |errorCallback|
-                with a newly created {{GeolocationPositionError}} object whose
-                {{GeolocationPositionError/code}} attribute is initialized to
-                {{GeolocationPositionError/PERMISSION_DENIED}}.
-                </li>
-                <li>Terminate this algorithm.
-                </li>
-              </ol>
+            context</a>, then [=call back with error=] |errorCallback| and
+            {{GeolocationPositionError/PERMISSION_DENIED}} and terminate this
+            algorithm.
+            </li>
+            <li>[=Request permission to use=] "geolocation" (see
+            [[[#privacy_for_uas]]] for permissions UI requirements).
+            </li>
+            <li data-tests="watchPosition_permission_deny.https.html">If the
+            <a>current settings object</a>'s [=environment settings object /
+            responsible document=] is not [=allowed to use=] the "geolocation"
```

Comment:
Similarly here, "request permission to use" will return "denied" if the current settings object isn't allowed to use geolocation.